litmus-portal/backend/auth/go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/litmuschaos/litmus/litmus-portal/backend/auth
 go 1.13
 
 require (
-	github.com/99designs/gqlgen v0.11.3
```

Comment:
Why are you removing "	github.com/99designs/gqlgen v0.11.3" and "go.mongodb.org/mongo-driver"

Reply:
Just did a go mod tidy as you asked..it automatically got removed.

Reply:
Okay, these changes are in auth